time start_time
uint16 msg_seq
uint16 original_num
float32 latitude_num
